moabb/datasets/Weibo2014.py

```diff
@@ -123,14 +139,15 @@ def _get_single_subject_data(self, subject):
                     'Pz', 'P2', 'P4', 'P6', 'P8', 'PO7', 'PO5', 'PO3', 'POz',
                     'PO4', 'PO6', 'PO8', 'CB1', 'O1', 'Oz', 'O2', 'CB2', 'VEO',
                     'HEO']
+        # fmt: on
```

Comment:
Example of a long list of `ch_names` keeping their formatting by using `fmt: on/off`

Reply:
Yes, this is a typical example where we want to turn off auto formatting.

Reply:
imo it would be nice to fomat it in more common way like this:
```python
ch_names = [
    'Fp1', 'Fpz', 'Fp2', 'AF3', 'AF4', 'F7', 'F5', 'F3', 'F1',
    'Fz', 'F2', 'F4', 'F6', 'F8', 'FT7', 'FC5', 'FC3', 'FC1',
    'FCz', 'FC2', 'FC4', 'FC6', 'FT8', 'T7', 'C5', 'C3', 'C1',
    'Pz', 'P2', 'P4', 'P6', 'P8', 'PO7', 'PO5', 'PO3', 'POz',
    'PO4', 'PO6', 'PO8', 'CB1', 'O1', 'Oz', 'O2', 'CB2', 'VEO', 'HEO',
]
```

Reply:
Yes, seems more congruent. @ErikBjare what do you think?

Reply:
More than that if we end up with double quotes it would be great to have them here too.

Reply:
Ok, commit done for this one. We